packages/katello/katello/backup.rb

```diff
@@ -331,8 +333,8 @@ def backup_config_files
       puts "Backing up config files... "
       flags = "--exclude=\"/var/www/html/pub/isos*\" --exclude=\"/var/www/html/pub/exports*\" --selinux --create --gzip"
       flags += " --file=#{File.join(@dir, 'config_files.tar.gz')} --listed-incremental=#{File.join(@dir, '.config.snar')}"
-      tar_command = "tar #{flags} #{configure_configs.join(' ')} 2>/dev/null"
-      run_cmd(tar_command, [0,2])
+      tar_command = "tar #{flags} #{configure_configs.join(' ')}"
```

Comment:
Is there a risk that this will now exceed the maximum command length? What about arguments with spaces? Does ```run_cmd``` handle an array as argument?

Reply:
Not more or less than before. The `Dir.glob()` above should only add a few more entries to the already super long line.

And no, `run_cmd` does not handle shell-specialchars properly.

Reply:
Previously it let the shell expand and I think that handles at least spaces. Moving it to Ruby moves the responsibility.

Reply:
We can do `all_configs.select { |path| Dir.glob(path).any? }` above, so just using `Dir.glob` for testing, not for actual resolution, this would keep the escaping of these files at shell level

Reply:
That's neat and I like it.